core/shared/src/main/scala/scodec/bits/crc.scala

```diff
@@ -49,6 +49,13 @@ object crc {
   lazy val crc32c: BitVector => BitVector =
     int32(0x1edc6f41, 0xffffffff, true, true, 0xffffffff).andThen(i => BitVector.fromInt(i))
 
+  /** An immutable "builder" to incrementally compute a CRC.
+    */
+  sealed trait CrcBuilder[Out] {
```

Comment:
I don't hold this opinion strongly but wdyt of using the names from collection library?
- rename `update` to `+=`
- rename `output` to `result`

Reply:
Eh on second thought, I don't like that suggestion as this is immutable.

Reply:
for sure, these definitely need bike-shedding :)

- 👍 `output` to `result`
- `update` to `+=`: maybe as an alias? it's a bit confusing since we're not exactly "adding", is there some other symbol that could be more appropriate? Also, since it's immutable, I think we'd want `+` so that `+=` works if they store it in a `var.

Reply:
Oops, just saw your followup comment. Yep!

Reply:
Btw, should `update` be `updated`? I think that's how they do immutable structures?

Reply:
👍 for `updated` and `result`

Reply:
Done and done!